subsys/net/CMakeLists.txt

```diff
@@ -4,4 +4,6 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+#find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 add_subdirectory(lib)
+add_subdirectory(l2_if_conn)
```

Comment:
This is only for Wi-Fi L2, so, the naming should reflect that, here and elsewhere.

Reply:
Changed it to l2_wifi_if_conn. Included "wifi" reference in other places as well.